src/parser.rs

```diff
@@ -362,9 +438,10 @@ impl Library {
             }
         }
 
-        if attrs.by_name("is-gtype-struct").is_some() {
-            return Ok(());
-        }
+        // TODO: unsure of how to handle this
+        //if attrs.by_name("is-gtype-struct").is_some() {
+        //    return Ok(());
+        //}
```

Comment:
Yes but it also needs an initialised Type to return. This is where I am unsure what to do... I could maybe return a `default()` initialised Type?